lib/aws_auditor/cache_instance.rb

```diff
@@ -4,13 +4,14 @@ module AwsAuditor
   class CacheInstance
     extend InstanceHelper
     extend CacheWrapper
+    extend AWSWrapper
 
     class << self
       attr_accessor :instances, :reserved_instances
     end
 
-    attr_accessor :id, :name, :instance_type, :engine, :count
-    def initialize(cache_instance)
+    attr_accessor :id, :name, :instance_type, :engine, :count, :tag_value
+    def initialize(cache_instance, account_id=nil, tag_name=nil, cache=nil)
```

Comment:
I think passing in account_id is extraneous, since this class extends AWSWrapper and can call its methods.